enterprise/models.py

```diff
@@ -975,6 +975,30 @@ def get_program(self, program_uuid):
             return None
         return CourseCatalogApiServiceClient(self.enterprise_customer.site).get_program_by_uuid(program_uuid)
 
+    def get_course_run_enrollment_url(self, course_run_key):
+        """
+        Return enterprise landing page url with the catalog information for the given course.
+
+        Arguments:
+            course_run_key (str): The course run id for the course to be displayed.
+        Returns:
```

Comment:
nit: add a new line above to separate `Arguments` section from `Returns` section.